### PR TITLE
[lldb] Add the experimental swift-enable-cxx-interop flag

### PR DESCRIPTION
**Pull request by augusto2112**
Add an experimental flag to turn on C++ interop in lldb.

### CODE REVIEW

**Comment by augusto2112** [comment]
@swift-ci test

**Review by adrian-prantl — APPROVED** [review]
This looks good; I'm going to not ask for a test because I assume there will be lots to follow?

File: lldb/source/Target/TargetProperties.td [hunk 1]
```diff
@@ -22,6 +22,9 @@ let Definition = "target_experimental" in {
   def SwiftEnableBareSlashRegex: Property<"swift-enable-bare-slash-regex", "Boolean">,
     DefaultFalse,
     Desc<"Passes the -enable-bare-slash-regex compiler flag to the swift compiler.">;
```
Comment (adrian-prantl) [inline lldb/source/Target/TargetProperties.td:24]: Can this be removed now, or is this still experimental?

Comment (kastiglione) [reply lldb/source/Target/TargetProperties.td:24]: last I looked it was still required…

**Pull request merged**